packages/vscode-extension/package.nls.json

```diff
@@ -267,6 +267,8 @@
     "teamstoolkit.localDebug.tunnelServiceNotStartedError": "The tunnel service hasn't been started. Please wait for a moment or restart the local tunnel task.",
     "teamstoolkit.localDebug.tunnelEndpointNotFoundError": "Failed to find the tunnel endpoint. Teams toolkit try to get the first HTTPS URL from %s and failed.",
     "teamstoolkit.localDebug.tunnelEnvError": "Failed to save environment variables. %s",
+    "teamstoolkit.localDebug.launchTeamsWebClientError": "Failed to launch Teams web client. %s",
```

Comment:
The %s placeholder will be quite confusing to localization team. If it's an independent string, separate it from this one.

Reply:
Remove %s for this case. Thanks for the suggestion.

Reply:
Just found that this string will be directly used in `package.json`, so just keep it.